lib/livebook_web/components/core_components.ex

```diff
@@ -597,6 +597,35 @@ defmodule LivebookWeb.CoreComponents do
     """
   end
 
+  @doc """
+  Renders a circular spinner.
```

Comment:
Ideally we don't want to inline icons like this because if they are sent several times on the same page they are duplicated but I think the risk of that happening is low here, right? If so, ship it. :)

Reply:
Yeah, sounds reasonable in this case :)